ScoutSuite/providers/azure/resources/resources.py

```diff
@@ -3,10 +3,14 @@
 from ScoutSuite.providers.base.configs.resources import CompositeResources
 
 
+# TODO: add docstrings.
 class AzureCompositeResources(CompositeResources, metaclass=abc.ABCMeta):
 
     async def _fetch_children(self, parent, **kwargs):
-        for child_class in self._children:
+        for child_class, name in self._children:
```

Comment:
Doesn't it throw a not enough values to unpack kind of error if there is no name?

Reply:
Oh, I see, you put 'None'.